test/acceptance/kafo_configure_test.rb

```diff
@@ -131,7 +130,7 @@ module Kafo
       end
 
       it 'must create file' do
-        skip 'Requires Puppet 4.5+ for data in modules' if Puppet::PUPPETVERSION < '4.5.0'
+        skip 'Requires Puppet 4.5+ for data in modules' if Gem::Specification.find_all_by_name('puppet').sort_by(&:version).last.version < Gem::Version.new('4.5.0')
```

Comment:
Ah ok, makes sense. I don't recall if I had a reason to write it like that in other tests. Refactoring would be nice but not required in this PR. I'll merge and do the release, thank you.